src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/models/rgw-bucket-encryption.ts

```diff
@@ -0,0 +1,7 @@
+export class RgwBucketEncryptionModel {
+  kmsProviders = ['vault'];
+  authMethods = ['token', 'agent'];
+  secretEngines = ['kv', 'transit'];
+  sse_s3 = 'AES256';
+  sse_kms = 'aws:kms';
```

Comment:
Also, do we need to assign these values in the model? Usually model will be just a `key` and then its type right?

Reply:
I think(maybe am wrong) values can be initialised in a model as well, so as in future if we want to add more values to these columns, we just need to do it here, no need to change anything in the form.